docs/01-overview/main-areas/eventing/evnt-02-useful-links.md

```diff
@@ -16,7 +16,7 @@ To learn more about how Eventing works, see:
 To perform tasks with Eventing, go through these tutorials:
 
 - [Tutorial: Trigger your workload with an event](../../../02-get-started/04-trigger-workload-with-event.md) - part of the [Get Started guides](../../../02-get-started), shows how to deploy a Function and trigger it with an event.
-- [Tutorial: Create subscription subscribing to multiple event types](../../../03-tutorials/00-eventing/evnt-02-subs-with-multiple-filters.md) - shows how to subscribe to one or more event types using the Kyma Subscription.
+- [Tutorial: Create Subscription subscribing to multiple event types](../../../03-tutorials/00-eventing/evnt-02-subs-with-multiple-filters.md) - shows how to subscribe to one or more event types using the Kyma Subscription.
 - [Tutorial: Event name cleanup in Subscriptions](../../../03-tutorials/00-eventing/evnt-03-type-cleanup.md) - explains how Eventing Services filter out non-alphanumeric character from event names.
```

Comment:
```suggestion
- [Tutorial: Event name cleanup in Subscriptions](../../../03-tutorials/00-eventing/evnt-03-type-cleanup.md) - explains how Kyma Eventing filters out non-alphanumeric character from event names.
```
I see you changed "Eventing Services" to "Eventing" in most occurrences, so I think it applies here too?